common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -270,12 +270,13 @@ def test_breadcrumbs_push_topic(self):
             css=".forum-nav-browse-menu-item[data-discussion-id='{}']".format(self.discussion_id)
         )
         self.assertTrue(topic_button.visible)
+
         topic_button.click()
 
         # Verify the thread's topic has been pushed to breadcrumbs
         breadcrumbs = self.thread_page.q(css=".breadcrumbs .nav-item")
-        self.assertEqual(len(breadcrumbs), 2)
-        self.assertEqual(breadcrumbs[1].text, "Test Discussion Topic")
+        self.assertEqual(len(breadcrumbs), 3)
```

Comment:
Actually in the past we had a [discussion](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/discussion/helpers.py#L115) that was added in advanced settings but we need an [inline discussion](https://github.com/edx/edx-platform/blob/308fd66435956256d332587b31b8691735c3428b/common/test/acceptance/tests/discussion/helpers.py#L122-L134) to make it cohorted. Perviously, breadcrumbs were like that All Topics > Test Discussion Topic. In this case length of breadcrumbs were 2. Now we have breadcrumbs like this All Topics > Verified-category > Verified-subcategory and length is 3.